build.gradle

```diff
@@ -129,6 +129,9 @@ dependencies {
     implementation 'org.wycliffeassociates:8woc2018-common'
     implementation 'org.wycliffeassociates:kotlin-resource-container'
     implementation 'com.github.WycliffeAssociates:jdenticon-kotlin:-SNAPSHOT'
+
+    //Atlassian commonmark (for rendering markdown)
+    implementation 'com.atlassian.commonmark:commonmark:0.12.1'
```

Comment:
Uses the BSD 2-clause license. Should be fine, but I'm not sure where/how we're keeping track of licenses.